client/modules/IDE/components/SketchList.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import SketchList from './SketchList';
+
+export default {
+  title: 'IDE/SketchList',
+  component: SketchList
+};
+
+export const Default = {
+  args: {
+    user: {
+      username: 'Joe Blogs',
+      authenticated: true
+    },
+    username: 'Joe Blogs',
+    loading: false,
+    sketches: [
+      {
+        id: '1',
+        name: 'Play Sketch',
+        createdAt: Date.now().toString(),
+        updatedAt: Date.now().toString()
+      }
+    ]
```

Comment:
I was wondering why these sketches don't show up in the list.  Again, it's because the component ignores this prop and gets the actual value from Redux.

Reply:
Ahhh! I was wondering that myself but was unsure.